### DISCUSSION TITLE
Keep previous version on gitstable when releasing new version

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#15537] [Wishlist]
Sometimes the release of a new version introduces new bugs. Sometimes they are specific to an installation and sometimes generic.

To make it easier to understand if an "issue" is a bug, it would be great if the previous version would be available on gitstable for some time (1 to 2 weeks after release) and the new release on gitdeveloper. This would allow to better identify issues by comparing the outcome on both systems (with identical data behind it).

**Comment by mariuszkrzaczkowski** [comment]
git stable is connected to github from the stable branch, it happens automatically
